### PR TITLE
Cut the cookies, added the code, and deleted the todos

### PR DESCRIPTION
**Pull request by dastels**

### CODE REVIEW

File: adafruit_logger.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+"""
+#pylint:disable=redefined-outer-name,consider-using-enumerate,no-self-use
+
+# imports
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_Logger.git"
+
+import time
```
Comment (kattni) [inline adafruit_logger.py]: Please place this import on line 45 and delete the `# imports` comment.

File: adafruit_logger.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        """
+        self._level = NOTSET
+        if handler is None:
+            self._handler = PrintHandler()
+        else:
+            self._handler = handler
+
+    @property
+    def level(self):
+        """Get the level."""
```
Comment (kattni) [inline adafruit_logger.py]: You can simply docstring this as `The level.`. We try to stay away from using "Get" and "Set".

File: adafruit_logger.py [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        return self._level
+
+    @level.setter
+    def level(self, value):
+        """Set the level."""
```
Comment (kattni) [inline adafruit_logger.py]: `setter`s do not need to be docstringed. You can remove this.

File: adafruit_logger.py [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+        self._level = value
+
+    def log(self, level, format_string, *args):
+        """Log a message.
+        level -- the priority level at which to log
```
Comment (kattni) [inline adafruit_logger.py]: Please use `:param level: The priority level at which to log` formatting for documenting keywords. Include whitespace above the first `:param` and below the last `:param`, but it is not necessary between `:params`.
e.g.
```
"""Log a message.

:param level: The priority of the level at which to log.
:param format_string: the core message string with embedded formatting directives
:param args: arguments ``format_string.format()``, can be empty

"""
```

File: adafruit_logger.py [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+        format_string -- the core mesage string with embedded formatting directives
```
Comment (kattni) [inline adafruit_logger.py]: `message` typo.

**Comment by jerryneedell** [comment]
There is an example in the Readme, that uses the UartHandler, but that is not in the code yet, correct, or am I totally missing something.... very possible

**Pull request merged**